src/DataFixtures/TransactionFixtures.php

```diff
@@ -33,6 +33,7 @@ public function load(ObjectManager $manager): void
             $transaction->setEducator($educator);
 
             $transaction->setAmount($item['amount']);
+            $transaction->setAccountNumber('165000112012133333');
```

Comment:
```suggestion
            $transaction->setAccountNumber('265104031000361092');
```

Koristi ovaj racun, posto imamo mod 97 validaciju pa mora da bude ispravan.